docs/images.md

```diff
@@ -143,6 +164,9 @@ Get options for ticket-id with read-only access using file backend:
       "max_writers": 1
     }
 
+Note that `size` and `format` are not reported, and this backend does
+not support multiple writers.
```

Comment:
I think this is goo enough for the legacy file backend. We don't want people to use it and we better not spend time on making it better. New features should be available only with the nbd backend.